Tempo/Views/ProgressView.swift

```diff
@@ -8,37 +8,48 @@
 
 import UIKit
 
+enum Type {
+	case NormalPlayer
+	case ExpandedPlayer
+}
+
 class ProgressView: UIView {
 	
-	var delegate: PostDelegate!
+	var playerDelegate: PostDelegate!
 	let fillColor = UIColor.tempoLightRed
 	private var updateTimer: NSTimer?
+	var indicator: UIView?
 	
 	override func drawRect(rect: CGRect) {
-		let progress = delegate.post?.player.progress ?? 0
+		var progress = 0.0
+		progress = playerDelegate.currentPost?.player.progress ?? 0
```

Comment:
lines 24-25 could be condensed to one line: `let progress = playerDelegate.currentPost?.player.progress ?? 0.0`